tasks/IKF.cpp

```diff
@@ -98,6 +98,9 @@ void IKF::imu_samplesTransformerCallback(const base::Time &ts, const ::base::sam
             if (correction_idx == correction_numbers)
             {
                 acc = correctionAcc / (double)correction_numbers;
+
+                acc_body = acc; // save this to be output later after being corrected
```

Comment:
The integration rate of the filtered acceleration measurement is, depending on the correction_numbers value, lower than the output rate. So currently the corrected acceleration measurement is written out with the same speed of the incoming samples, but the value changes only if a new measurements is integrated. I would either use every incoming sample or write the filtered acceleration only when it is integrated, but then the frequency of orientation samples and acceleration samples would be different.